packages/webapp/pages/squads/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactElement, useContext } from 'react';
+import { NextSeoProps } from 'next-seo/lib/types';
+import { NextSeo } from 'next-seo';
+import { BaseFeedPage } from '@dailydotdev/shared/src/components/utilities';
+import useSidebarRendered from '@dailydotdev/shared/src/hooks/useSidebarRendered';
+import classNames from 'classnames';
+import { SOURCES_QUERY } from '@dailydotdev/shared/src/graphql/sources';
+import InfiniteScrolling, {
+  checkFetchMore,
+} from '@dailydotdev/shared/src/components/containers/InfiniteScrolling';
+import { useInfiniteQuery } from 'react-query';
+import request from 'graphql-request';
+import { graphqlUrl } from '@dailydotdev/shared/src/lib/config';
+import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
+import {
+  FeedContainer,
+  SourceCard,
+  SquadListingHeader,
+} from '@dailydotdev/shared/src/components';
+import EditIcon from '@dailydotdev/shared/src/components/icons/Edit';
+import { IconSize } from '@dailydotdev/shared/src/components/Icon';
+import { mainFeedLayoutProps } from '../../components/layouts/MainFeedPage';
+import FeedLayout, { getLayout } from '../../components/layouts/FeedLayout';
+import { defaultOpenGraph, defaultSeo } from '../../next-seo';
+
+const seo: NextSeoProps = {
+  title: `Squads`,
```

Comment:
```suggestion
  title: `Squad directory`,
```

Need to ensure it adds it to output as:
```
Squad directory | daily.dev
```